package.yaml

```diff
@@ -41,7 +41,7 @@ dependencies:
   - aeson >=1.0 && <1.5
   - aeson-better-errors >=0.8
   - aeson-pretty
-  - ansi-terminal >=0.7.1 && <0.9
```

Comment:
We probably added these because we broke cabal's solver. Try to relax them far enough to include the newer versions rather than fully removing the bounds.

Reply:
What is the command to test the cabal solver ?

Reply:
I would run `stack sdist`, extract the produced tarball into a temporary directory, change into that directory, and then run `cabal update && cabal build`. Unfortunately this doesn't quite work right now because of the `purescript-cst` and `purescript-ast` package splits as those packages haven't been uploaded to Hackage yet. I don't know if `cabal` supports multi-project setups like stack does. If the package split issue is difficult to resolve it may be best to just wait until after we have released 0.14 before worrying about updating.